iml-gui/crate/src/components/action_dropdown/confirm_action_modal.rs

```diff
@@ -76,6 +80,8 @@ pub fn update(msg: Msg, model: &mut Model, orders: &mut impl Orders<Msg, GMsg>)
         }
         Msg::StateChangeSent(_) => {
             //@TODO: Open command modal here
+            log!("TODO: Open command modal here, StateChangeSent");
+            model.ready_to_command = true;
```

Comment:
This is the workaround.

Reply:
Thinking about this a bit, the command modal is really a singleton. The user will only have a single one open at a time. 

Due to this, We could just embed a `command_modal` at the top-level of the app and use GMsg to send an open command to it.